tests/tests.src.js

```diff
@@ -6620,6 +6620,11 @@
 			Error,
 			'Parse error: forbidden code point when `allowUnsafeSymbols: true` and `strict: true`'
 		);
+		equal(
+			he.encode('äöüÄÖÜ', { 'decimalOutput': true}),
```

Comment:
Please use escapes sequences here, matching the other tests: [`'\xE4\xF6\xFC\xC4\xD6\xDC'`](https://mothereff.in/js-escapes#1%C3%A4%C3%B6%C3%BC%C3%84%C3%96%C3%9C)